releases/release-1.22/release-notes/maps/pr-101742-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 101742
+releasenote:
+  text: Enable MaxSurge for `DaemonSet` by default.
```

Comment:
Should it be text: Enable `MaxSurge` for `DaemonSet` by default ?

Reply:
Yep, maxSurge is an option in the daemonset.spec.updateStrategy.rollingUpd

Reply:
Updated. Thanks for the review